### DISCUSSION TITLE
Change user in PKC

### DISCUSSION THREAD

**Discussion by Kybeer** [croneter/PlexKodiConnect#1779] [Ideas]
Hello,
Is there any smart way to switch Plex user in PKC? I'm talking about the "local" Plex profiles, not the main online Plex-accrount (that controls your license etc)

I have three users in my home, and the only way I have found to switch user is to reboot the device (a Raspberry Pi) where Kodi+PKC is running.

**Comment by croneter** [comment]
https://github.com/croneter/PlexKodiConnect/wiki/Several-Plex-Users

**Reply by Kybeer** [reply]
Thanks @croneter , I will take a look.

But I'm more looking at a way to "simply" make the initial PKC login screen appear with the Managed Users, without rebooting - rather than full fledged Kodi profiles.

**Reply by croneter** [reply]
There's the PKC setting logging you out: navigate to Add-ons -> Video Add-ons -> PlexKodiConnect -> "Log out user XYZ". Maybe you can bind that to a key or something?

**Reply by Kybeer** [reply]
Yes, but as soon as I try to bind that to a menu item, it actually just logs me off (takes me to the initial login screen) - instead of letting me configure it. 

If there's anyway to prevent that behavior, it would be the perfect solution.

**Reply by croneter** [reply]
No, that's by design. It will log you off and you will need to manually select the new Plex user. There's no way to directly switch to an explicit other Plex user

**Reply by Kybeer** [reply]
Thanks for the clarification @croneter . Binding it to a key is perhaps not so easy then.

**Reply by hamakavula** [reply]
> Hello, Is there any smart way to switch Plex user in PKC? I'm talking about the "local" Plex profiles, not the main online Plex-accrount (that controls your license etc)
> 
> I have three users in my home, and the only way I have found to switch user is to reboot the device (a Raspberry Pi) where Kodi+PKC is running.

Fairly simple: You can create a menu/favourite/shortcut and switch user with ease, use this custom string/command:
`ActivateWindow(10025,&quot;plugin://plugin.video.plexkodiconnect/?mode=switchuser&quot;,return)`

Example (edit in userdata/favourites.xml)
```
<favourites>
    <favourite name="Log-out Plex user" thumb="special://home/addons/plugin.video.plexkodiconnect/icon.png">ActivateWindow(10025,&quot;plugin://plugin.video.plexkodiconnect/?mode=switchuser&quot;,return)</favourite>
</favourites>
```

I have it working on all my devices like that now (Kodi 19.3 + PKC 3.6.14)
